crates/utils/re_byte_size/src/arrow_sizes.rs

```diff
@@ -13,17 +13,24 @@ impl SizeBytes for dyn Array {
     }
 }
 
-impl SizeBytes for ArrayRef {
+impl SizeBytes for &dyn Array {
     #[inline]
     fn heap_size_bytes(&self) -> u64 {
         self.get_array_memory_size() as u64
     }
 }
 
-impl<T: ArrowPrimitiveType> SizeBytes for PrimitiveArray<T> {
+impl<T: Array> SizeBytes for &T {
```

Comment:
Method already takes `&self`:
```suggestion
impl<T: Array> SizeBytes for T {
```

Reply:
I can't: 

![image](https://github.com/user-attachments/assets/6971e0f4-ffa7-41bf-98de-c747b9dfd803)